release_notes/unreleased.md

```diff
@@ -1 +1,2 @@
 **Unreleased**
+* Initial Release with Python 3 support
```

Comment:
the PR has more release note points we should probably include here

Reply:
@rgil-splunk , The mentioned release notes contain information about all the actions as it is a new app. But, this information is already available in the app documentation. Also, we are keeping the best practice to add "Initial Release with Python 3 support" as the release notes for new apps.